pkgs/development/python-modules/hcipy/default.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  setuptools,
+  setuptools-scm,
+
+  # dependencies
+  numpy,
```

```suggestion
  numpy_1,
```
Package is not compatible with 2.x, which will be the new default very soon.